testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/ClientPoliciesImportExportTest.java

```diff
@@ -57,6 +58,13 @@ public void beforeAbstractKeycloakTestRealmImport() {
         removeAllRealmsDespiteMaster();
     }
 
+    @After
+    @Override
+    public void after() throws Exception {
+        testingClient.testing().exportImport().clear();
+        super.after();
```

Comment:
This call could lead to double calling of `super.after()` since both this and `super.after()` is annotated with `@After`.

To be less error-prone I'd suggest renaming this method to some other non-colliding name and left without calling `super.after()`.

Reply:
Out of curiosity I did a local debugging test: With my original change, the method was called only once. Even after removing my `@After` annotation and relying on the annotation in the parent class worked and it was called once. So it seems that JUnit caught up with what one would call a sensible behavior. 

Still, I changed the code as you suggested and pushed it. The tests are again green. Ready for a review.

Reply:
Thanks for checking this! It's an example of a better-safe-than-sorry code, and the change improved it so it cannot be executed twice even if JUnit would accidentally change the behaviour in the future.